tasks/_inc_client_build_steps.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Install packages with yarn
+  yarn:
+    executable: "yarn --network-timeout 300000 --check-files"
+    path: "{{ galaxy_server_dir }}/client"
+  environment:
+    PATH: "{{ galaxy_venv_dir }}/bin:{{ ansible_env.PATH }}"
+    VIRTUAL_ENV: "{{ galaxy_venv_dir }}"
+
+- name: Run gulp
+  command: gulp
+  args:
+    chdir: "{{ galaxy_server_dir }}/client"
+  environment:
+    PATH: "{{ galaxy_server_dir }}/client/node_modules/.bin:{{ galaxy_venv_dir }}/bin:{{ ansible_env.PATH }}"
+    VIRTUAL_ENV: "{{ galaxy_venv_dir }}"
+    NODE_ENV: production
+
+- name: Run webpack
+  command: webpack -p
```

Comment:
Same as above; `yarn run webpack-production` which will apply appropriate flags (right now just -p, but who knows down the road)